CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+v0.6.0
+======
+
+In #204 :
+
+- Require PHP 7.1 as minimum (previous php versions are EOL)
+- PHP 7.3 support
+- Update Gaufrette dep to v0.8
+- Update Symfony deps to v4
+
+v0.5.3
+======
+
+- Fix composer.json validity #184
+- Be able to install Gaufrette v0.6 #185
+
+v0.5.2
+======
+
+- Declare command as service to fix sf3.4 deprecation #183<Paste>
```

Comment:
Could you remove `<Paste>` please?

Reply:
thanks for the review, changes are done ;)